### ISSUE TITLE
Serrated incisors

### ISSUE THREAD

**Issue by drhasanbas** [obophenotype/human-phenotype-ontology#7506]
**Preferred term label:**
Serrated incisors

**Synonyms**
Jagged incisors

**Definition (free text, please give PubMed ID)**
Incisor teeth with irregular edges resembling saw (PMID: 25439729)

**Parent term (use hpo.jax.org/app)**
Abnormal incisor morphology (HP:0011063)

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Filippi syndrome (OMIM: #272440)

**Your nano-attribution (ORCID)**
0000-0003-3475-564X

**Comment by pnrobinson** [comment]
Thanks, I added the term and will add the annotation. The data should be visible with the next HPO release in June.
